3.10/release-notes-new-features310.md

```diff
@@ -299,6 +299,28 @@ for details.
 
 This feature is only available in the Enterprise Edition.
 
+### Optimization rule `arangosearch-constrained-sort`
+
+This new optimization rule brings significant performance improvements by 
+allowing you to perform sorting and limiting inside an ArangoSearch View
```

Comment:
Does this equally apply to both View types?
Please refer to them like this: `arangosearch` Views, `search-alias` Views ("ArangoSearch Views" is ambiguous now that we have two types, possibly referring to both as part of the larger ArangoSearch feature, or referring to the traditional `arangosearch` Views).